### PR TITLE
[node] Implement GeoJSON converter

### PR DESCRIPTION
**Pull request by brunoabinader**
This is needed in order to let functions like `addSource` being used via style operators in render tests.

### CODE REVIEW

**Comment by asheemmamoowala** [comment]
@brunoabinader - is this the same as : #5623?

**Comment by brunoabinader** [comment]
@asheemmamoowala I think so - currently we have a _generic_ GeoJSON converter in a sense that it accepts stringified JSON in `std::string` form. It is the base for e.g. Qt and Android converters. In this PR I add the same functionality to Node.

**Comment by brunoabinader** [comment]
> Ideally, we'd be converting the v8 object to our internal GeoJSON representation directly instead of stringifying

The intent is to reuse the same string-based converter already used by other platforms. I'd prefer to use this instead of implementing a whole new converter from scratch.

File: platform/node/src/node_geojson.hpp [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
 template <>
-optional<GeoJSON> Converter<GeoJSON>::operator()(const v8::Local<v8::Value>&, Error& error) const {
-    error = { "not implemented" };
-    return {};
+optional<GeoJSON> Converter<GeoJSON>::operator()(const v8::Local<v8::Value>& value, Error& error) const {
+    v8::Local<v8::Value> local = value;
+
+    Nan::HandleScope scope;
+
+    v8::Local<v8::Object> JSON = Nan::To<v8::Object>(
```
Comment (jfirebaugh) [inline platform/node/src/node_geojson.hpp]: Use [`Nan::JSON.Stringify`](https://github.com/nodejs/nan/blob/master/doc/json.md#api_nan_json_stringify).

File: platform/node/test/ignores.json [hunk 2]
```diff
@@ -41,10 +41,9 @@
   "render-tests/regressions/mapbox-gl-js#3548": "skip - needs issue",
   "render-tests/regressions/mapbox-gl-js#3682": "https://github.com/mapbox/mapbox-gl-js/issues/3682",
   "render-tests/regressions/mapbox-gl-native#7357": "https://github.com/mapbox/mapbox-gl-native/issues/7357",
-  "render-tests/runtime-styling/image-add-sdf": "skip - https://github.com/mapbox/mapbox-gl-native/issues/9847",
```
Comment (jfirebaugh) [inline platform/node/test/ignores.json]: There's probably a few other tests than can be enabled.

File: platform/node/src/node_geojson.hpp (hunk 1 above)
Comment (brunoabinader) [reply platform/node/src/node_geojson.hpp]: Good call - we can use it [here](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/node/src/node_map.cpp#L154-L165) as well.

File: platform/node/test/ignores.json (hunk 2 above)
Comment (brunoabinader) [reply platform/node/test/ignores.json]: Yes - `render-tests/regressions/mapbox-gl-js#3548`.

**Pull request merged**